packages/documentation/docs/user-guide/installation/initial-sofie-core-setup.md

```
@@ -1,12 +1,12 @@
 ---
-sidebar_position: 3
+sidebar_position: 30
 ---
 
 # Initial Sofie Core Setup
 
 #### Prerequisites
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)